### DISCUSSION TITLE
Minutes 27 June

### DISCUSSION THREAD

**Discussion by zorp** [standard-for-public-code/standard-for-public-code#1039] [Community calls]
## Proposal agenda points
*  Vote on adding two security settings noticed missing from the GH-organisation
* Discuss ongoing distunguish actions between Foundation and Standard
* Guidelines for translations
*  Further Discuss domain register, and github-verification
* Can we close the criteria PR fix? Demo with the fix
* Claus has implemented the just-read-the-docs theme. Vote on this?

## Attendees
Josef, Johan, Claus and Rasmus

### Vote on adding two security settings noticed missing from the GH-organisation

* How many votes is needed to decide something? There is agreement that it depend on the change. Things that can be easily reverted can be decided by the people present in a community call. But the decision making process is described in the governance file for the standard, https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/GOVERNANCE.md. There is agreement that the described process is sufficient.
* It is proposed to enable two-factor authentication as a requirement for users? Everyone agree to turn it on as a requirement.
* Proposal in regards of personal access tokens. We should enable the use of fine grained personal access tokens and restrict the use of the classic way. Everyone agree.

### Discuss ongoing distinguish actions between Foundation and Standard

* It needs to be clear that the standard is upstream and the foundation version is downstream.
It is decided to create a PR for Foundation to mention they are a downstream with link to upstream. I'm uncertain who got the responsibility, but I think @jgroenen catched this one.

### Guidelines for translations

* Currently there is a repro https://github.com/standard-for-public-code/community-translations-standard and there is a guide for creating a print PDF https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/docs/printing.md. @jgroenen  will look into the guide in connection with creating a Dutch translation.

### Further Discuss domain register, and github-verification

* It is proposed to register a domain for the upstream (community) version. The following domains are discussed: _http://standardforpubliccode.community/_ or _http://standard.publiccode.community/_ or _http://standardforpubliccode.org/_. There is agreement about using **standardforpubliccode.org**. @jgroenen  will complete the domain registration.

### Can we close the criteria PR fix (#1036)? Demo with the fix

The pull-request have been reviewed and can be merged. @zorp will merge the PR #1038.

### Claus has implemented the just-read-the-docs theme. Vote on this?

To distinguish the standard from the foundation look'n'feel it is proposed to change the template to Just the Docs. There is agreement on the change and move to just-the-docs. @clausmullie  will take action.

**Comment by Ainali** [comment]
Did you decide a date for the next meeting? (I imagine that July was cancelled due to vacations.)

**Reply by clausmullie** [reply]
Yes indeed, everyone was on holiday. They are set for last Thursday of every month, 29/8 being the next one

**Reply by os2git** [reply]
I see that I don't have anything in my calendar for last Thursday in the month. @clausmullie are you the one owning the meeting invite?

**Reply by zorp** [reply]
Ups wrong account ... os2git = zorp :)

**Reply by clausmullie** [reply]
No I believe the event is owned by an @publiccode.net account - @mcarlson could you confirm?
